client/static/js/board.js

```diff
@@ -1,26 +1,233 @@
 
-var Board = {
-    edges: null,
-    maxRows: 1000,
-    maxCols: 1000,
-
-    init: function(maxRows, maxCols) {
-        this.edges = [];
-        maxRows = maxRows || 1000;
-        maxCols = maxCols || 1000;
-        return this;
-    },
+var board = {
+    edges: [],
+    canvas : document.createElement("canvas"),
+    curx : 200,
+    cury : 200,
+    cursor: new edge(0,0,0,0,'yellow'),
+    init : function() {
+            this.canvas.width =  window.innerWidth;
+            this.canvas.height =  window.innerHeight;
+            this.context = this.canvas.getContext("2d");
+            document.body.insertBefore(this.canvas, document.body.childNodes[0]);
+
+            this.frameNo = 0;
+            this.interval = setInterval(updateboard, 20);
+
```

Comment:
Does this need to `return this;` since we are storing `board.init()` in the other file?